gdx/src/com/badlogic/gdx/math/Intersector.java

```diff
@@ -498,112 +498,105 @@ public static boolean intersectRaySphere (Ray ray, Vector3 center, float radius,
 	}
 
 	/** Intersects a {@link Ray} and a {@link BoundingBox}, returning the intersection point in intersection. This intersection is
-	 * defined as the point on the ray closest to the origin which is within the specified bounds.
+	 * defined as the point where the ray hits the box.
 	 * 
 	 * <p>
 	 * The returned intersection (if any) is guaranteed to be within the bounds of the bounding box, but it can occasionally
 	 * diverge slightly from ray, due to small floating-point errors.
 	 * </p>
-	 * 
-	 * <p>
-	 * If the origin of the ray is inside the box, this method returns true and the intersection point is set to the origin of the
-	 * ray, accordingly to the definition above.
-	 * </p>
+	 *
 	 * @param intersection The intersection point (optional)
 	 * @return Whether an intersection is present. */
 	public static boolean intersectRayBounds (Ray ray, BoundingBox box, Vector3 intersection) {
-		if (box.contains(ray.origin)) {
```

Comment:
better keep original behavior (and related documentation) instead of introducing an unnecessary breaking change.